extension/json/src/functions/table_functions/json_scan.cpp

```diff
@@ -719,23 +720,13 @@ struct JsonScanBindData : public ScanFileBindData {
 };
 
 uint64_t JsonScanBindData::getFieldIdx(const std::string& fieldName) const {
-    auto normalizedName = fieldName;
-    // TODO(Ziyi): this is a temporary fix for the json testing framework, since copy-to-json
-    // always outputs the property name, so we have to remove the property name prefix for
-    // matching.
-    if (!colNameToIdx.contains(fieldName)) {
-        std::regex pattern(R"(^[^.]+\.(.*))");
-        std::smatch match;
-        if (std::regex_match(fieldName, match, pattern) && match.size() > 1) {
-            normalizedName = match[1]; // Return the part of the string after the "xxx."
-        }
-    }
-    return colNameToIdx.contains(normalizedName) ? colNameToIdx.at(normalizedName) : UINT64_MAX;
+    auto itr = colNameToIdx.find(fieldName);
+    return itr == colNameToIdx.end() ? UINT64_MAX : itr->second;
```

Comment:
I thought it's easier to use `colNameToIdx.contains() and .at()` but up to u

Reply:
The std::unordered_map is much faster than using contains and at.